var/spack/repos/builtin/packages/aml/package.py

```diff
@@ -9,60 +9,122 @@
 
 
 class Aml(AutotoolsPackage):
-    """AML: Building Blocks for Memory Management."""
+    """AML is a memory management library designed to ease the use of complex
```

Comment:
Is there a reason not to inherit from `CudaPackage` or `RocmPackage`?

Reply:
We don't depend on the compilers, we only need to link to the underlying libraries. I don't mind inheriting from both Cuda and ROCm but:
- the cuda_arch variant is not going to be used by the library
- it's unclear how I can use ROCmPackage to allow a hip variant running on top of the nvidia runtime
- the ROCmPackage is going to force dependencies that we don't need